### PR TITLE
Update sdlb 2.6 extended, remove Polynote

### PR DESCRIPTION
**Pull request by zzeekk**
Major update and refactoring:
- Updated to latest SDLB version
- Polynote & Derby Metastore is no longer part of the getting started, but a separate repository "polynote-lab". sparkShell.sh should be used instead.
- Scripts for standard operations (buildSpark.sh, startJob.sh...), only Linux/Podman.
- Mock data mode for CustomWebserviceDataObject by setting mockJsonDataObject configuration.
- Github Workflow extended with exporting config and schema/stats.

### CODE REVIEW

**Review by iliassoto — COMMENTED** [review]
Thanks for the updates @zzeekk and @dust629

File: config/application.conf [hunk 1]
```diff
@@ -1,21 +1,78 @@
...
+    # shared local metastore (metastore must be started)
+    #"spark.hadoop.javax.jdo.option.ConnectionURL" = "jdbc:derby://localhost:1527/db;create=true"
+    #"spark.hadoop.javax.jdo.option.ConnectionDriverName" = "org.apache.derby.jdbc.ClientDriver"
+    #"spark.hadoop.javax.jdo.option.ConnectionUserName" = "sa"
+    #"spark.hadoop.javax.jdo.option.ConnectionPassword" = "1234"
+    # shared local s3 storage service for bucket 'data' (s3proxy must be started)
+    #"spark.hadoop.fs.s3a.bucket.data.impl" = "org.apache.hadoop.fs.s3a.S3AFileSystem"
+    #"spark.hadoop.fs.s3a.bucket.data.access.key" = "admin"
+    #"spark.hadoop.fs.s3a.bucket.data.secret.key" = "1234"
+    #"spark.hadoop.fs.s3a.bucket.data.endpoint" = "http://s3:8888/"
```
[the remark this answers is not in the document]
Comment (zzeekk) [reply config/application.conf:17]: True, i removed it.

File: config/application.conf [hunk 2]
```diff
@@ -1,21 +1,78 @@
-# This is the application.conf file that you are meant to modify during this tutorial.
-# In the beginning, it has the same contents as application.conf.part-1-solution so that you can test if your local environemnt is setup correctly.
+
+# This is the application.conf file as it should look after completing part-3
```
[the remark this answers is not in the document]
Comment (zzeekk) [reply config/application.conf:2]: True, i cleaned it up.

[the remark this answers is not in the document]
Comment (zzeekk) [reply config/application.conf:2]: True, i cleaned it up.

File: config/application.conf [hunk 3]
```diff
@@ -1,21 +1,78 @@
...
+    }
+    # enable to use mock-data as fallback if webservice is down
+    #mockJsonDataObject = stg-departures-mock
   }
-  stg-departures {
+  # data object with mock data used by ext-departures, if webservice is not available
+  stg-departures-mock {
     type = JsonFileDataObject
-    path = "~{id}"
+    path = ${env.basePath}fallback-download/stg-departures
```
[the remark this answers is not in the document]
Comment (zzeekk) [reply config/application.conf]: Yes we could, but maybe its easier like it is now? to be discussed (in another PR?)

**Pull request merged**